# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/svg/SVGSVGElement.h
# Commit: b848ca90888e
# Full Hash: b848ca90888ee8e446f3d54b65c3ef68c71a1451
# Author: longsonr <longsonr@gmail.com>
# Date: 2019-06-20 22:06:31
# Regressor Bug: 1551030
# File Overlap Count: 2
# Description:
#   Bug 1551030 - Merge all SVGRect classes r=dholbert
# ==============================================================================

diff -r 20956a783e08 -r b848ca90888e dom/svg/SVGSVGElement.h
--- a/dom/svg/SVGSVGElement.h	Thu Jun 20 14:19:57 2019 +0300
+++ b/dom/svg/SVGSVGElement.h	Thu Jun 20 07:03:54 2019 -0700
@@ -25,7 +25,7 @@
 class DOMSVGLength;
 class DOMSVGNumber;
 class SVGMatrix;
-class SVGIRect;
+class SVGRect;
 class SVGSVGElement;
 
 // Stores svgView arguments of SVG fragment identifiers.
@@ -137,7 +137,7 @@
   already_AddRefed<DOMSVGAngle> CreateSVGAngle();
   already_AddRefed<nsISVGPoint> CreateSVGPoint();
   already_AddRefed<SVGMatrix> CreateSVGMatrix();
-  already_AddRefed<SVGIRect> CreateSVGRect();
+  already_AddRefed<SVGRect> CreateSVGRect();
   already_AddRefed<DOMSVGTransform> CreateSVGTransform();
   already_AddRefed<DOMSVGTransform> CreateSVGTransformFromMatrix(
       SVGMatrix& matrix);